### PR TITLE
Use bulk metadata endpoint on members page

### PR DESCRIPTION
**Pull request by cnasc**
# Description

This PR updates the members page to use the bulk metadata endpoint to get all keyholder information in a single pass.

image
(screenshot proof that it works as before)

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [x] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [x] My code follows the style guidelines of this project, including naming conventions
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/__tests__/reducers/metadataReducer.test.ts [hunk 1]
```diff
@@ -31,7 +47,7 @@ describe('metadataReducer', () => {
     expect(result).toBe(initialState)
   })
 
-  it('should return the initial state when receiveing SET_NETWORK', () => {
+  it('should return the initial state when receiving SET_NETWORK', () => {
```
Comment (julien51) [inline unlock-app/src/__tests__/reducers/metadataReducer.test.ts]: thanks!

File: unlock-app/src/__tests__/reducers/metadataReducer.test.ts [hunk 2]
```diff
@@ -53,20 +69,11 @@ describe('metadataReducer', () => {
...
-          color: 'blue',
-        },
-      },
-    })
+    const result = reducer(undefined, gotBulkMetadata(lockAddress, metadata))
 
     expect(result).toEqual({
-      'my address': {
-        '7': {
+      [lockAddress.toLowerCase()]: {
```
Comment (julien51) [inline unlock-app/src/__tests__/reducers/metadataReducer.test.ts]: haha! I see how this is biting you too ;p

File: unlock-app/src/components/content/MembersContent.tsx [hunk 3]
```diff
@@ -80,11 +80,10 @@ const MetadataTableWrapper = ({
   })
 
   useEffect(() => {
-    // Dispatch request for key metadata here, only when data changes
+    // Dispatch requests for key metadata here, only when data changes
```
Comment (julien51) [inline unlock-app/src/components/content/MembersContent.tsx]: again! thanks.

File: unlock-app/src/__tests__/reducers/metadataReducer.test.ts (hunk 2 above)
Comment (cnasc) [reply unlock-app/src/__tests__/reducers/metadataReducer.test.ts]: Took me longer than I care to admit to realize why my filter was coming back empty

**Pull request merged**